src/fastboot-app-server.js

```diff
@@ -163,28 +174,46 @@ class FastBootAppServer {
     let env = this.buildWorkerEnv();
     let worker = cluster.fork(env);
 
+    let firstBootResolve;
+    let firstBootReject;
+    const firstBootPromise = new Promise((resolve, reject) => {
+      firstBootResolve = resolve;
+      firstBootReject = reject;
+    });
```

Comment:
This seems a bit odd, can you refactor to not capture the resolve/reject and instead move the `worker.on` code inside the promise callback?